jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/VirtualThreads.java

```diff
@@ -125,15 +125,29 @@ public static boolean isVirtualThread()
      * @param namePrefix the prefix to use for the name of the virtual threads
      * @return a virtual threads {@code Executor} that will name the virtual threads according to the provided name prefix.
      */
-    public static Executor getNamedVirtualThreadsExecutor(String namePrefix)
+    public static ThreadFactoryExecutor getNamedVirtualThreadsExecutor(String namePrefix)
```

Comment:
This would be a breaking change (e.g. Spring).
Why is this necessary?

Reply:
I cannot find any code at spring-projects that use the method `getNamedVirtualThreadsExecutor`

https://github.com/search?q=org%3Aspring-projects+jetty+getNamedVirtualThreadsExecutor&type=code

This includes spring-boot, spring-framework, examples, etc ...

Reply:
@joakime see discussion in #11353.